docs/root/intro/arch_overview/upstream/circuit_breaking.rst

```diff
@@ -3,6 +3,14 @@
 Circuit breaking
 ================
 
+.. attention::
+
+  Circuit breaking behavior changed in version 1.14. If the previous behavior (documented `here <https://www.envoyproxy.io/docs/envoy/v1.13.0/intro/arch_overview/upstream/circuit_breaking>`_
+  is needed, disable runtime feature "envoy.reloadable_features.new_http1_connection_pool_behavior" or
+  "envoy.reloadable_features.new_http2_connection_pool_behavior" and then re-configure your clusters or
+  restart Envoy. Note that the old behavior is deprecated and will be removed according to the deprecation
+  policy.
```

Comment:
My feeling is to just kill this attention block. The new behavior is documented below and we can document the changes in the release notes? WDYT?

Reply:
I think that's fine.  A new user will come to this page, read the docs, and set the correct circuit breaker settings. An upgrader whose config no longer works will probably check release notes first.